Kitodo/src/main/resources/messages/errors_de.properties

```
@@ -122,7 +124,7 @@ processTitleEmpty=Kein Vorgangstitel angegeben.
 processTitleInvalid=Ung\u00FCltiger Titel f\u00FCr einen Vorgang. Er solltet mit dem regul\u00E4ren Ausdruck ''{0}'' \u00FCbereinstimmen.
 projectTitleAlreadyInUse=Der Projekttitel wird bereits verwendet.
 propertiesForProcessNotSaved=Eigenschaften des Prozesses {0} konnten nicht gespeichert werden.
-prozesskopieForm.createNewProcess.noInsertionPositionSelected=Es wurde keine Position f�r die Titelsatzverkn\u00FCpfung ausgew\u00E4hlt.
+prozesskopieForm.createNewProcess.noInsertionPositionSelected=Es wurde keine Position f�r die Titelsatzverkn\u00FCpfung ausgew\u00E4hlt.
```

Please use correct escaped German umlaut ü
```suggestion
prozesskopieForm.createNewProcess.noInsertionPositionSelected=Es wurde keine Position f\u00FCr die Titelsatzverkn\u00FCpfung ausgew\u00E4hlt.
```